Part A - Basics/PYTHON_A.adoc

```diff
@@ -961,17 +927,22 @@ include::08_functions/examples/func_variable_arguments_python3.py[]
 
 
 === Unpacking Arguments
-Unpacking arguments are arguments that are unpacked from a list or tuple. They are used when you
-want to pass the elements of a list or tuple as arguments to a function. Unpacking arguments are
-specified using the `\*` operator in the function call. The elements of the list or tuple are
+Arguments that are unpacked from a list or tuple are used when you
+want to pass the elements of a list or tuple as arguments to a function. Unpacking arguments is done using the `*` operator in the function call. The elements of the list or tuple are
 assigned to the parameters in the function signature.
 
 [source,python]
 ----
 include::08_functions/examples/func_unpacking_arguments.py[]
 ----
 
-=== Variable Scope
+=== Namespace and Scope
+
+By creating a variable and assigning value to it, you create a key-value pair like in a dictionary.
```

Comment:
Rejected

Please move the name space section to chapter 2, part 10 and revert to the old name `Variable Scope`.